src/masternode-payments.cpp

```diff
@@ -205,7 +205,7 @@ bool IsBlockPayeeValid(const CTransaction& txNew, int nBlockHeight, CAmount bloc
     return true;
 }
 
-void FillBlockPayments(CMutableTransaction& txNew, int nBlockHeight, CAmount blockReward, CTxOut& txoutMasternodeRet, std::vector<CTxOut>& voutSuperblockRet)
+void FillBlockPayments(CMutableTransaction& txNew, int nBlockHeight, CAmount blockReward, std::vector<CTxOut>& txoutsMasternodeRet, std::vector<CTxOut>& voutSuperblockRet)
```

Comment:
`std::vector<CTxOut>`s have `vout` prefix in code currently (`vout` in `CTransaction` and `voutSuperblockRet` in mnpayments/governance/blocktemplate), let's keep it consistent maybe i.e. s/`txoutsMasternodeRet`/`voutMasternodeRet`/ or (even more descriptive) s/`txoutsMasternodeRet`/`voutMasternodePaymentsRet`/ ?